deploy/bundle/README.md

```diff
@@ -0,0 +1,60 @@
+# Bundling the Windows Machine Config Operator
+This directory contains resources related to installing the WMCO onto a cluster using OLM.
+
+## Pre-requisites
+This guide assumes [OLM](https://github.com/operator-framework/operator-lifecycle-manager) has been installed to the
+cluster, and [operator-sdk](https://github.com/operator-framework/operator-sdk) and
+[opm](https://github.com/operator-framework/operator-registry/) have been installed on localhost.
+
+## Creating a bundle image
+A bundle image can be created by editing the CSV in deploy/bundle/windows-machine-config-operator/manifests/
+and replacing `$OPERATOR_IMAGE` with the location of the operator image you wish to deploy.
+You can then run the following command in the root of this git repository
+```
+operator-sdk bundle create $BUNDLE_REPOSITORY:$VERSION_TAG -d deploy/bundle/windows-machine-config-operator/manifests \
+--channels alpha --default-channel alpha --image-builder podman
+```
+The variables in the command should be changed to match the container image repository you wish to store the bundle in.
+You can also change the channels based on the release status of the operator.
+
+You should then push the image to the remote repository
+```
+podman push $BUNDLE_REPOSITORY:$BUNDLE_VERSION
+```
+
+You should verify that the new bundle is valid:
+```
+operator-sdk bundle validate $BUNDLE_REPOSITORY:$BUNDLE_VERSION --image-builder podman
+```
+
+## Creating a new operator index
+An operator index is a collection of bundles. Creating one is required if you wish to deploy your operator on your own
+cluster.
+
+```
+opm index add --bundles $BUNDLE_REPOSITORY:$VERSION_TAG --tag $INDEX_VERSION
+```
+
+### Editing an existing operator index
+An existing operator index can have bundles added to it:
+```
+opm index add --from-index $INDEX_VERSION
+```
+and removed from it:
+```
+opm index rm
+```
+
+## Deploying the operator on a local cluster
+Change catalogsource.yaml to point to the operator index created above. Now deploy it:
+```
+oc apply -f catalogsource.yaml
+```
+This will deploy a CatalogSource object in your current namespace. You can check the status of it via:
+```
+oc describe catalogsource
+```
+Now wait 1-10 minutes for the catalogsource's status.connectionState.lastObservedState field to be set to READY.
+Once this is done, log into the cluster's web console, then go to the OpenShift GUI console. In 'Administration' mode,
+go to the Operators/OperatorHub tab. Search for the 'Windows Machine Config Operator' and click install.
+The operator will then be installed onto your cluster.
```

Comment:
Given you have been doing everything in the command line, please do the same here rather than using the GUI. This will also help with the follow up testing story when a script would need to be written. You can have the GUI section as an alternate deployment section. 

I am remember the bundle needing a `package.yaml`. Has that requirement been removed?

Reply:
I wasn't aware you could do it via CLI. I'll figure out how.

The package.yaml requirement has been removed. Heres an example of the new way to bundle operators: https://github.com/operator-framework/bundle-example/tree/master/bundle/manifests